### ISSUE TITLE
ESP32S2-TFT SPI conflict with pin 36

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#5843]
### CircuitPython version

```python
7.2.0-alpha.1
```

### Code/REPL

```python
Adafruit CircuitPython 7.2.0-alpha.1 on 2021-12-29; Adafruit Feather ESP32-S2 TFT with ESP32S2
>>> import board
>>> spi = board.SPI()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: D36 in use
>>>
```

### Behavior

see above

### Additional information

Thsi works OK on 7.1.0

**Comment by ladyada** [comment]
this is that thing with the stuff @jepler 😓  :D

**Comment by jerryneedell** [comment]
Fixed by #5846 
using bus.SPI(...) still repost SCK in use
but board.SPI() works

**Comment by jepler** [comment]
`bus.SPI(...)` is expected not to work, because the pins are in use by the TFT.  I think it's a bug in 7.1.x that it seemed to work.

`board.SPI()` is (with #5846) the exact same SPI object used by the display, so they can both use it without conflicting over the pins.
